src/main/java/cloud/eppo/callback/CallbackManager.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Map;
 import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Consumer;
+import org.jetbrains.annotations.NotNull;
```

Comment:
At first I was thinking why have JetBrains bleed into the source, but then I remembered Android Studio is based off JetBrains 😅

Reply:
It's just a little harmless documentation and contract declaration